tools/ci/pr_preview.py

```diff
@@ -256,6 +267,32 @@ def delete_ref(self, refspec):
             full_ref
         ])
 
+    def get_pr_open_numbers(self):
```

Comment:
It looks like `pr` is rarely, if ever, used in public method names. For consistency, maybe `get_open_pull_requests` and document the return value is a list of numbers?

Reply:
Yeah, I struggled to convey this is only looking at the refs in the remote, and doesn't touch github. Switched to `get_pull_requests_with_open_ref`, wdyt?

Reply:
SGTM